pymc/distributions/continuous.py

```diff
@@ -1901,7 +1901,7 @@ class StudentT(Continuous):
     rv_op = studentt
 
     @classmethod
-    def dist(cls, nu, mu=0, lam=None, sigma=None, *args, **kwargs):
+    def dist(cls, nu=1, mu=0, sigma=None, lam=None, *args, **kwargs):
```

Comment:
I would instead take the default nu from the HalfStudentT, I don't see a good reason to provide a default.

Reply:
Hence the need to document the defaults in the docstring!